CHANGELOG.next.asciidoc

```diff
@@ -20,7 +20,8 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 
 *Filebeat*
- - [Httpjson]- Fix for pagination at root level not working when used with chaining.
+- [Httpjson]- Fix for pagination at root level not working when used with chaining.
```

Comment:
Does this need a newline separating Filebeat section and bulleted list? The github preview does not render as I would expect, but I'm not sure if that affects our website.

Reply:
It probably wants more than that. Propose
```suggestion

- Fix HTTPJSON pagination at root level when used with chaining. {issue}32618[32618] {pull}32722[32722]
```

Reply:
That looks good.